chapter07/kds/ex4.pl

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "Enter some text or supply an input file as an argument:\n";
+while( <> ){
+  print "$_" if( /[A-Z][a-z]/ );
```

Comment:
Actually, it would match the "Bo", part of the title, so ignore me. But then, it wouldn't match if the only capitalised letter was the last letter (of which I'm struggling to think of an example) ?